packages/wallets/src/features/cfd/screens/CompareAccounts/CompareAccountsCard.scss

```diff
@@ -0,0 +1,35 @@
+.wallets-compare-accounts-main-container {
+    padding: 0 1rem;
+}
+
+.wallets-compare-accounts-card-container {
```

Comment:
I think with BEM, top level class should be consistent with component name.
So for example:
.wallets-compare-accounts-card {
  // then block which is inside; if that makes sense for given case, its better to have names after function/role of specific div, instead of generic and meaningless "wrapper" "container" (though its not always possible)

}